### DISCUSSION TITLE
Unexpected error while building project from inside the jest

### DISCUSSION THREAD

**Discussion by hopeless-programmer-online** [vercel/next.js#21183] [Help]
# Bug report

## Describe the bug

`An unexpected error has occurred` in browser while starting server built inside the jest test.

## To Reproduce

1. Create new next.js project.
2. Add jest test with `execSync('npx next build')` call (or so).
3. Run the test to perform build.
4. Run `next start`.
5. See `An unexpected error has occurred` in browser.

_(can provide a repo if needed)_

## Expected behavior

The behavior should be the same as for the builds performed from outside the jest (terminal, script, etc.).

## Screenshots

![Screenshot_2020-12-28  An unexpected error has occurred](https://user-images.githubusercontent.com/63446812/103191369-69818400-48dd-11eb-93b9-30fae3c537c3.png)

## System information

- OS: Windows 7
- Browser: Firefox 84.0
- Version of Next.js: 10.0.4
- Version of Node.js: 14.15.1

## Additional context

I'm trying to achieve end-to-end testing for my package. It contains react+typescript icons so I though it would be a good idea to reproduce the whole workflow with it (from the perspective of the end user) inside the test.
Basically I do the following:
1. Create dummy package (next.js project) with dependencies that includes my package.
2. Run 'next build` and `next start` from *inside* the jest test.
3. Get error listed above.

In the very end the problem is based around `next build`.
If it was performed from jest then `next start` will result in an error, no matter where I run it (jest, regular script, terminal).
If `next build` was performed from outside the jest (regular script, terminal) then there will be no error even in jest.

_(90% sure it is a jest problem but still think it would be much faster to find the root cause from this side)_
